docs/how-to/deploy.md

```diff
@@ -0,0 +1,46 @@
+---
+id: deploy
+title: Amplication Deployment
+sidebar_label: Overview
+slug: /deploy
+---
+
+# Amplication Deployment
+
+Amplication provides flexible and powerful deployment options for services generated using our platform.
+This guide provides an overview of Amplication's deployment options and links to specific instructions for various deployment targets.
+
+## Understanding Amplication's Deployment Architecture
+
+Applications created with Amplication comprise two main components:
```

Comment:
That is not always true. There could be other components like a Kafka message broker, or a Redis cache or others - they can all be additional containers that would be specified in the docker-compose file. Those two are the minimum, there could be more.